package.json

```diff
@@ -136,6 +136,7 @@
       "global": {
         "lines": 40
       }
-    }
+    },
+    "testTimeout": 30000
```

Comment:
Needed for e2e. Eventually when we split up e2e and unit, we can keep this in e2e only. We may need to increase as our e2e tests get more complicated and take longer.